test/letters/containers/LettersApp.unit.spec.jsx

```diff
@@ -2,18 +2,57 @@ import React from 'react';
 import SkinDeep from 'skin-deep';
 import { expect } from 'chai';
 
-import LettersApp from '../../../src/js/letters/containers/LettersApp';
+import { LettersApp, AppContent } from '../../../src/js/letters/containers/LettersApp';
 
 import reducer from '../../../src/js/letters/reducers';
 import createCommonStore from '../../../src/js/common/store';
 
 const store = createCommonStore(reducer);
 
+const mockRoutes = [{ path: '/fake' }];
+const defaultProps = {
+  store,
+  location: { pathname: '/blah' },
+  route: { childRoutes: mockRoutes },
+  profile: {
+    userFullName: { first: null, middle: null, last: null, suffix: null },
+    email: null,
+    dob: null,
+    gender: null,
+    accountType: null,
+    terms: { loading: false, terms: {} },
+    savedForms: [],
+    prefillsAvailable: [],
+    loading: true
+  }
+};
+
 describe('<LettersApp>', () => {
-  it('should render', () => {
-    const mockRoutes = [{ path: '/fake' }];
-    const tree = SkinDeep.shallowRender(<LettersApp store={store} location={{ pathname: '/blah' }} route={{ childRoutes: mockRoutes }}/>);
-    const vdom = tree.getRenderOutput();
-    expect(vdom).to.not.be.undefined;
+  it('should render AppContent', () => {
+    const tree = SkinDeep.shallowRender(<LettersApp {...defaultProps}/>);
+    // div() throws an exception if it can't find the selector
+    expect(tree.dive(['AppContent'])).to.not.be.null;
```

Comment:
Since `dive()` throws if it doesn't find the selector, would something like `expect(() => tree.dive(['AppContent'])).to.not.throw()` be more closely aligned with the assertion? Probably not worth messing with.

Reply:
Yeah, it's debatable, honestly. If it does throw, the test will fail, so I did it this way because I think it's a little more obvious what we're actually testing.